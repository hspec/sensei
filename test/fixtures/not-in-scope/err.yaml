version: '1.0'
ghcVersion: ghc-9.10.0
span:
  file: test/fixtures/not-in-scope/Foo.hs
  start:
    line: 2
    column: 7
  end:
    line: 2
    column: 16
severity: Error
code: 88464
message:
- 'Variable not in scope: catMaybes'
hints: []
