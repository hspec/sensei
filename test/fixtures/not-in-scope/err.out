test/fixtures/not-in-scope/Foo.hs:2:7: error: [GHC-88464]
    Variable not in scope: bar

